app/rnx2rtkp/rnx2rtkp.c

```diff
@@ -154,17 +154,15 @@ int main(int argc, char **argv)
         else if (!strcmp(argv[i],"-d")&&i+1<argc) solopt.timeu=atoi(argv[++i]);
         else if (!strcmp(argv[i],"-b")) prcopt.soltype=1;
         else if (!strcmp(argv[i],"-c")) prcopt.soltype=2;
-        else if (!strcmp(argv[i],"-i")) prcopt.modear=2;
-        else if (!strcmp(argv[i],"-h")) prcopt.modear=3;
+        else if (!strcmp(argv[i],"-i")) prcopt.modear=ARMODE_INST;
+        else if (!strcmp(argv[i],"-h")) prcopt.modear=(prcopt.modear==ARMODE_WL?ARMODE_WL_FIXHOLD:ARMODE_FIXHOLD);
```

Comment:
i think we can define another separate character for ARMODE_WL_FIXHOLD, if "-w" is parsed later than "-h", then `prcopt.modear` would be `ARMODE_FIXHOLD`.
For example we can do `strcmp(argv[i],"-z" prcopt.modear=ARMODE_WL_FIXHOLD`, then in the `strcmp(argv[i],"-w"`, we only need to do `prcopt.modear=ARMODE_WL`.